src/mmw/js/src/modeling/models.js

```diff
@@ -170,6 +192,29 @@ var ScenarioModel = Backbone.Model.extend({
 
         this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove', this.attemptSave, this);
+
+        var debouncedPoll = _.debounce(_.bind(this.pollForResults, this), 500);
+        this.get('modifications').on('add change', function() {
+            debouncedPoll();
+        });
+        this.set('taskModel', $.extend(true, {}, App.currProject.get('taskModel')));
```

Comment:
Since we can now get `taskModel` from project it seems redundant to store task model on the scenarios as well. Consider removing this property from scenarios.